### DISCUSSION TITLE
support for owncloud?

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#327] [Q&A]
Exchanging with @kennedycasey and @marisacasillas, I realized that the most lightweight thing for the ACLEW users would be to use their current owncloud-organized repo for the long-form audios. @lucasgautheron do you know whether this is possible, and if so, how would one go about it?

**Comment by lucasgautheron** [comment]
Hello,

It is theoretically possible: https://git-annex.branchable.com/tips/owncloudannex/
The easiest way would be through webdav (https://git-annex.branchable.com/special_remotes/webdav/).
More information about using special remotes with datalad can be found here: http://handbook.datalad.org/en/latest/basics/101-138-sharethirdparty.html#the-common-case-repository-hosting-without-annex-support-and-special-remotes

Performance was not great on our nextcloud instance through webdav, but I think the issue was more on our side... Besides, it could be that owncloud is much more efficient.

I can explain in further details if needed.
